main.py

```diff
@@ -1,5 +1,32 @@
+
+import random
 from IPython.display import clear_output
 
+def choose_first():
+    '''
+    This is function five from the notebook
+    It chooses randomly which player goes first
+    '''
+    player = random.randint(1,2)
+    if player == 1:
```

Comment:
Maybe consider using:
`print(f'{player} shopuld start')`
instead of using conditions.